blinkit/page-2.html

```diff
@@ -0,0 +1,191 @@
+                    <div>-</div>
+                    <div>2</div>
+                    <div>+</div>
+                </div>
+            </div>
+            <div class="items-added-to-cart">
+                <div class="item-added-to-cart">
+                    <div class="item-detail-cart">
+                        <div class="item-image"><img src="images/23383a.jpg"></div>
+                        <div class="item-purchase-details">
+                            <p>Local Carrot</p>
+                            <p>500g</p>
+                            <p>₹17 <span style="text-decoration: line-through; color: grey;">₹19</span></p>
+                        </div>
+                    </div>
+                    <div class="quantity-selector-cart">
+                        <div>-</div>
+                        <div>3</div>
+                        <div>+</div>
+                    </div>
+                </div>
+            </div>
+            <div class="items-added-to-cart">
+                <div class="item-added-to-cart">
+                    <div class="item-detail-cart">
+                        <div class="item-image"><img src="images/391306a.avif"></div>
+                        <div class="item-purchase-details">
+                            <p>Onion</p>
+                            <p>1kg</p>
+                            <p>₹32 <span style="text-decoration: line-through;font-size: 14px; color: grey;">₹41</span></p>
+                        </div>
+                    </div>
+                    <div class="quantity-selector-cart">
+                        <div>-</div>
+                        <div>1</div>
+                        <div>+</div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+
+    </section >
+
+    <section class="payment-section">
+        <div class="payment-details">
+            <div>MRP</div>
+            <div>₹176</div>
+        </div>
+        <div class="payment-details">
+            <div>Product discount</div>
+            <div style="color:  rgb(12, 131, 31); font-size: 15px; font-weight: 700" >-₹33</div>
+        </div>
+
+        <div class="payment-details">
+            <div>Delivery Charge</div>
+            <div style="color:  rgb(12, 131, 31); font-size: 15px; font-weight: 700"><span style="text-decoration:line-through">₹15</span> FREE</div>
+
+        </div>
+        <br>
+        <br>
+        <div class="payment-details">
+            <div class="grand-total"  >Grand Total</div>
+            <div class="grand-total">₹143</div>
+        </div>
+
+        <div class="footer-text" id="footer-text-page2" >Coupons are onlly applicable on the Blinkit app</div>
+
+        <div class="total-bar">
+            <div>
+                6 items . ₹143 <span style="text-decoration: line-through;">₹176</span>
+            </div>
+            <div>
+                Proceed >
+            </div>
+        </div>
+
+
+    </section>
+
+
+    <section class="footer-section">
```

Comment:
We have named this section as footer-section but it is not the part of a footer
Please name it accordingly